.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Thank you for reporting an issue.
+
+Please fill in as much of the template below as you're able.
+-->
+
+## Version
+
+<!--
+List the versions of all `tokio` crates you are using. The easiest way to get
+this information is using `cargo-tree`.
+
+`cargo install cargo-tree`
+(see install here: https://github.com/sfackler/cargo-tree)
+
+Then:
+
+`cargo tree | grep tokio`
```

Comment:
`$ cargo tree -i -p tokio` - show the version of `tokio` crate only
`$ cargo tree -p tokio` - show all `tokio` crates

Reply:
Happy to take improvements as a follow up PR.